src/java.management/share/classes/java/lang/management/MemoryMXBean.java

```diff
@@ -267,6 +267,30 @@ public interface MemoryMXBean extends PlatformManagedObject {
      */
     public MemoryUsage getNonHeapMemoryUsage();
 
+    /**
+     * Returns the CPU time used by garbage collection.
+     *
+     * <p> This is the CPU time used by all garbage collection
+     * activity, including any overhead, which means the result
+     * may be non-zero even if no GC has occurred.
+     *
+     * This method returns {@code -1} if the platform does
+     * not support this operation or the information is not
+     * available.
+     *
+     * @implNote Reported time will include relevant
+     * implementation-specific details such as driver threads,
+     * workers, VM Operations and string deduplication (if enabled).
+     * The return value can be -1 if called when measurement is
+     * not possible, such as during shutdown.
```

Comment:
For the implNote then I think you are looking to say something HotSpot VM specific. That is possible with something like:  The specifics on what constitutes the time spent in garbage collection is highly implementation dependent. In the HotSpot Virtual Machine implementation, ... 

(If you are using the term "driver threads" then you will probably need to say more on what it means).

It is still the intention to use this in conjunction with OperatingSystemMXBean:: getProcessCpuTime ? If so, this could also be included here, or in an API note.